interval.py

```diff
@@ -0,0 +1,333 @@
+import abc
```

Comment:
Please remove `interval.py`, as it should not need to be copied directly into the repository.